src/screenControl.py

```diff
@@ -296,12 +296,15 @@ def __init__(self, flags, keyBindings, stdscr, lineObjs, cursesAPI):
             if not lineObj.isSimple():
                 self.lineMatches.append(lineObj)
 
-        self.numLines = len(lineObjs.keys())
-        self.numMatches = len(self.lineMatches)
```

Comment:
had to move these because we should call `toggleSelectAll()` before assigning these, and need to call `resetDirty()` before calling `toggleSelectAll()`